plugins/qFacets/cellsFusionDlg.ui

```diff
@@ -299,9 +305,7 @@ Output facet normals may be randomly oriented
    <header location="global">ccOctreeSpinBox.h</header>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="qFacets.qrc"/>
```

Comment:
For some reason qtdesigner removed this. It was not intended but it did that by its own.